app/managers.py

```diff
@@ -0,0 +1,32 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self):
+        self._connection = sqlite3.connect("D:/for prog/library_db/cinema.db3")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str):
+        self._connection.execute(f"INSERT INTO {self.table_name} (first_name, last_name) VALUES (?, ?)",
+                                 (first_name, last_name)
+                                 )
+        self._connection.commit()
+
+    def all(self):
+        actor_cursor = self._connection.execute(f"SELECT id, first_name, last_name FROM {self.table_name}")
```

Comment:
Use select * because when you change Actor model you also must change Manager. And it will be unlogical